frontend/src/lib/api.js

```diff
@@ -56,17 +56,17 @@ class Api {
         }
         return await getJSONOrThrow(response)
     }
-    async create(url, data) {
+    async create(url, data, isFormData = false) {
```

Comment:
Nit: `update` and `create` are essentially the same code with a different HTTP method, could be abstracted?

Reply:
Probably. That's outside the scope of this PR though.

Ideally we would also have a TS-friendly API that will give the right types to all responses, e.g. get a response with a `UserType` if fetching `api/user`.... But again not inside this PR :)